tests/config/__snapshots__/default-config-staging.test.js.snap

```diff
@@ -25,7 +24,7 @@ Object {
   "domainName": "scp-staging.biomage.net",
   "emailDomainName": "https://ui-default.scp-staging.biomage.net",
   "pipelineIgnoreSSLCertificate": false,
-  "pipelineInstanceConfigUrl": "https://raw.githubusercontent.com/hms-dbmi-cellenics/releases/master/staging/pipeline.yaml",
+  "pipelineInstanceConfigUrl": "https://raw.githubusercontent.com/biomage-org/releases/master/staging/pipeline.yaml",
```

Comment:
All these snaps contain references to `biomage-org` because the default `githubOrganisationName` is  `biomage-org` (as well as a few hardcoded values in `tests/config/default-config-staging.test.js`). Is this a requirement so that the deployment works with all Biomage's private deployments or can some of this organisation name spread be eliminated?

Reply:
Maybe moving everything domain specific into `getDomainSpecific()` we can just mock this function and make the returns be domain agnostic